12.11_k번째_수.js

```diff
@@ -0,0 +1,11 @@
+function solution(array, commands) {
+    let arr2 =[]
+        for(let num1 = 0; num1 <= commands.length-1; num1++){
+    let arr1 = array.slice(commands[num1][0]-1,commands[num1][1])
+    arr1.sort(function(a,b){
+        return a - b;
+    })
```

Comment:
```suggestion
    arr1.sort((a,b) => {
        return a - b;
    })
```
```()=>{}``` ```arrow function```활용하면 좋습니다.

[참고사이트](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/Arrow_functions)